### PR TITLE
Added implementation for exposing the topics

### PR DESCRIPTION
**Pull request by marian-craciunescu**
WIP...More test should be added.

### CODE REVIEW

File: server/fcm/connector.go [hunk 1]
```diff
@@ -198,22 +199,51 @@ func (conn *Connector) GetPrefix() string {
...
+			http.Error(w, `{"error":"invalid parameters in request"}`, http.StatusBadRequest)
+			return
+		}
 		conn.deleteSubscription(w, topic, userID, fcmID)
+	case http.MethodGet:
+		conn.retriveAllSubscription(w, userID, fcmID)
+	}
+}
+
+func (conn *Connector) retriveAllSubscription(w http.ResponseWriter, userID, fcmID string) {
```
Comment (Bogh) [inline server/fcm/connector.go]: Rename to `retrieveSubscriptions` maybe?

File: server/fcm/connector.go [hunk 2]
```diff
@@ -198,22 +199,51 @@ func (conn *Connector) GetPrefix() string {
...
+	topics := make([]string, 0)
+
+	for k, v := range conn.subscriptions {
+		logger.WithField("key", k).Info("retriveAllSubscription")
+		if v.route.Get(applicationIDKey) == fcmID && v.route.Get(userIDKey) == userID {
+			logger.WithField("path", v.route.Path).Info("retriveAllSubscription path")
+			topics = append(topics, string(v.route.Path))
+		}
+	}
+	serialized, err := json.Marshal(topics)
```
Comment (Bogh) [inline server/fcm/connector.go]: If interested you can use `json.Encoder` to directly write in a Writer the encoding.
See the `Encoder` doc in `encoding/json` package: https://godoc.org/encoding/json

File: server/fcm/connector.go [hunk 3]
```diff
@@ -248,9 +278,7 @@ func (conn *Connector) deleteSubscription(w http.ResponseWriter, topic, userID,
 	fmt.Fprintf(w, `{"unsubscribed":"%v"}`, topic)
 }
 
-// parseParams will parse the HTTP URL with format /fcm/:userid/:fcmid/subscribe/*topic
-// returning the parsed Params, or error if the request is not in the correct format
-func (conn *Connector) parseParams(path string) (userID, fcmID, topic string, err error) {
+func (conn *Connector) parseUserIdAndDeviceId(path string) (userID, fcmID, unparsedPath string, err error) {
```
Comment (Bogh) [inline server/fcm/connector.go]: If ok rename to `getParams` or rename to use `ID` with caps at least.

File: server/fcm/connector.go (hunk 1 above)
Comment (marian-craciunescu) [inline server/fcm/connector.go]: done

File: server/fcm/connector.go (hunk 3 above)
Comment (marian-craciunescu) [inline server/fcm/connector.go]: done

**Pull request merged**